src/binder/bind/bind_ddl.cpp

```diff
@@ -137,8 +138,9 @@ std::vector<PropertyNameDataType> Binder::bindPropertyNameDataTypes(
                     propertyNameDataType.first.c_str()));
         }
         StringUtils::toUpper(propertyNameDataType.second);
-        boundPropertyNameDataTypes.emplace_back(
-            propertyNameDataType.first, Types::dataTypeFromString(propertyNameDataType.second));
+        auto dataType = Types::dataTypeFromString(propertyNameDataType.second);
```

Comment:
add a bindDataType function and refactor l82-83 together with l141-142. Also other places where you use `dataTypeFromString` under binder

Reply:
Added a new bindDataType func